docs/index.html

```diff
@@ -0,0 +1,133 @@
+<html>
+	<head>
+		<title>Grapher Documentation</title>
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+		<script type="text/javascript" src="https://vuejs.org/js/vue.js"></script>
+		<script type="text/javascript" src="https://unpkg.com/marked@0.3.6"></script>
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+		<script type='text/javascript'>
+			window.onload = function(){
+				marked.setOptions({
```

Comment:
Can you please properly indent the code ? Use prettier or something. And spaces instead of tabs